mesonbuild/coredata.py

```diff
@@ -418,24 +464,53 @@ def set_buildtype_from_others(self):
             mode = 'custom'
         self.builtins['buildtype'].set_value(mode)
 
-    def get_all_compiler_options(self):
-        # TODO think about cross and command-line interface. (Only .build is mentioned here.)
-        yield self.compiler_options.build
+    _V = TypeVar('_V')
 
-    def _get_all_nonbuiltin_options(self):
-        yield self.backend_options
-        yield self.user_options
-        yield from self.get_all_compiler_options()
-        yield self.base_options
+    @staticmethod
+    def get_prefixed_options_per_machine(
+        options_per_machine # : PerMachine[Dict[str, _V]]]
+    ) -> Dict[str, _V]:
```

Comment:
__[Flake8]__ [F821] undefined name '_V' ([link](https://sider.review/gh/repos/19784232/pulls/5263#issue-3981247))

<sub>posted by Sider</sub>

Reply:
I'm not sure what to do about this? `flake8` is just wrong, the class member is defined just above. (and yes I'm developing with `3.5`.)